proof/records.go

```diff
@@ -380,6 +380,9 @@ func GenesisRevealRecord(genesis **asset.Genesis) tlv.Record {
 
 func GroupKeyRevealRecord(reveal *asset.GroupKeyReveal) tlv.Record {
 	recordSize := func() uint64 {
+		// TODO(ffranr): Do we need dispatch on record size? I think
```

Comment:
No, this is only required for encoding. For decoding, the length is encoded in the record (the L in TLV). So can just panic here (which we usually do in these record size functions).

Reply:
Is this the generic version? if so, it should dynamically determine the size just by encoding.